_blueprints/developers_guide.adoc

```diff
@@ -0,0 +1,17 @@
+= Developer's Guide
+
+== Adding new packages
+
+ALMighty uses https://glide.sh[Glide] for package management.  To
+install a new package, you can follow these steps.  The `<URL>` should
+be replaced with the package URL pointing to Github or any other
+location.
+
+[source, bash]
+glide get <URL>
+glide update
+git add glide.yaml glide.lock
+git commit -m "<message>"
+
+NOTE: If the `glide get` commands prompts for versions range, most of the
+time you must select `M` (minor).
```

Comment:
There's no **rule** to select `M` but it is what makes the most sense I guess if you want to receive non-breaking updates from a package.